app/views/facility_orders/_merge_order_form.html.haml

```diff
@@ -1,10 +1,12 @@
 - if @merge_orders.blank?
   .well
     = simple_form_for [current_facility, @order], method: :put, html: { class: "js--edit-order" } do |f|
-      = text_field_tag :product_add_quantity, 1
+      = text_field_tag :product_add_quantity, 1, class: "js--edit-order__quantity"
```

Comment:
Is the double underscore a convention in the codebase?

Reply:
We use it in a couple places, mostly for CSS. It's definitely a convention in many of our newer projects.